tmd/areas/create_area_weights.py

```diff
@@ -24,23 +28,13 @@
 GFFILE_PATH = STORAGE_FOLDER / "output" / "tmd_growfactors.csv"
 POPFILE_PATH = STORAGE_FOLDER / "input" / "cbo_population_forecast.yaml"
 
-DUMP_LOSS_FUNCTION_VALUE_COMPONENTS = True
-OPTIMIZE_RATIOS = True  # True produces much better optimization results
-# wide bounds settings:
-# OPTIMIZE_LO_BOUND_RATIO = 2e-6  # must be greater than 1e-6
-# OPTIMIZE_HI_BOUND_RATIO = 9e+9  # must be less than np.inf
-# narrow bounds settings:
-# OPTIMIZE_LO_BOUND_RATIO = 1e-2  # must be greater than 1e-6
-# OPTIMIZE_HI_BOUND_RATIO = 1e2  # must be less than np.inf
-# tiny bounds settings:
-# OPTIMIZE_LO_BOUND_RATIO = 1e-1  # must be greater than 1e-6
-# OPTIMIZE_HI_BOUND_RATIO = 1e+1  # must be less than np.inf
-OPTIMIZE_LO_BOUND_RATIO = 0.0
-OPTIMIZE_HI_BOUND_RATIO = np.inf
-OPTIMIZE_FTOL = 1e-10
+DUMP_TARGET_LOSS_FUNCTION_VALUE_COMPONENTS = True
+REGULARIZATION_LAMBDA = 1e-9
+OPTIMIZE_FTOL = 1e-8
+OPTIMIZE_GTOL = 1e-8
 OPTIMIZE_MAXITER = 5000
-OPTIMIZE_VERBOSE = 0  # set to zero for no iteration information
-OPTIMIZE_RESULTS = False  # set to True to see complete lsq_linear results
+OPTIMIZE_IPRINT = 20  # set to zero for no iteration information
```

Comment:
Good default.

Reply:
Yes,  iprint = 20 is much better than iprint = 1 when you want diagnostic information.
But, setting iprint = 0 produces much cleaner output when doing production work.